nas.py

```diff
@@ -181,6 +181,28 @@ def get_control_default_route(self, host_name):
               .format(host_name))
         sys.exit(1)
 
+    def get_middle_box_left_iface_name(self, host_name):
```

Comment:
this can be encoded in a generic way, the "right" (or "left") is one parameter! "to-alpha" for example. Fix required